standards-catalogue/source/layouts/standards_table.erb

```diff
@@ -1,36 +1,52 @@
 <% wrap_layout :layout do %>
 
   <div style = "overflow-x:auto;">
-    <% if !current_page.data.title.nil? %>
-      <h1><%= current_page.data.title %></h1>
-    <% else %>
-      <h1><%= current_page.data.name %></h1>
-    <% end %>
+    <h1><%= current_page.data.title %></h1>
     <table style="width:100%">
-      <% current_page.data.each do |key, value| %>
-        <tr>
-          <td nowrap><%= key.titleize.capitalize %>: </td>
-          <% if key == 'link' %>
-            <td nowrap><strong><a href='<%= value %>'><%= value %></a></strong> </td>
-          <% elsif key == 'organisation' %>
-            <td nowrap><strong><%= display_organisation(value) %></strong> </td>
-          <% elsif key == 'status' %>
-            <td nowrap><strong><%= display_status(:standard, value) %></strong> </td>
-          <% elsif key == 'licence_id' %>
-            <%= display_licence(value) %>
-          <% elsif key == 'tags' %>
-            <td nowrap>
-              <ul>
-                <% current_page.data.tags.each do |tag| %>
-                  <li><strong><%= tag %></strong></li>
-                <% end %>
-              </ul>
-            </td>
-          <% else %>
-            <td nowrap><strong><%= value %></strong> </td>
-          <% end %>
-        </tr>
-      <% end %>
+      <tr>
+        <td>Name</td>
+        <td>
+          <strong><%=link_to current_page.data.name, current_page.data.specification_url %></strong>
+        </td>
+      </tr>
+
+      <tr>
+        <td>Also Known As</td>
+        <td>
+          <strong><%= current_page.data.contraction %></strong>
+        </td>
+      </tr>
+
+      <tr>
+        <td>Topics</td>
+        <td>
+          <ul>
```

Comment:
Are we happy with the indentation being off compared to the other entries in the table?
![2021-11-15-153034_702x307_scrot](https://user-images.githubusercontent.com/3315059/141808800-2f151bf7-8b39-48c4-a8ba-7fd4e6e844a4.png)

Reply:
should now look like:

![Screenshot 2021-11-15 at 16 02 58](https://user-images.githubusercontent.com/13121570/141814339-f3700de3-3e98-4130-986a-4f20750af3bc.png)